tox.ini

```diff
@@ -0,0 +1,50 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+skipsdist = True
+envlist =
+    py{35,36,37,py3}
+    style
+    coverage
+skip_missing_interpreters = true
+
+[testenv]
+description = run tests
+passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*
+deps =
+    -rrequirements.txt
```

Comment:
Typo?

Reply:
No, this is passed to pip as is and is a valid argument. (Like `ssh -p2202 ...`)